textile.go

```diff
@@ -416,7 +417,6 @@ func startNode() error {
 						cmd.Green(desc) + cmd.Grey(" update to thread "+thrd)
 					fmt.Println(msg)
 				}
-			default:
```

Comment:
will crush your fan!

Reply:
nice catch 👍. that was there because otherwise it'll block until an update comes... but with a buffer of 10 like you added, this should probably be ok now right?

Reply:
For me anyway, as simple as channels seem, they are weirdly unintuitive. In this case, we want the select to block. There's nothing else to do in the go routine besides wait for values. The default case is useful when reading / writing _a single_ value off an un-buffered channel that may not have a corresponding reader / writer, and is therefore blocking... so the default lets you sorta skip over it. Here though, the default is just causing the program to re-do the for loop in an unbounded manner, maxing cpu. 

As for the _the channel itself blocking_... A channel blocks _writes_ when it's not buffered or when it's buffer is full. So, as long as there's a listener, i.e., something pulling values out of the channel (like we're doing here) writes won't be blocked. The buffer ensure we don't _have_ to have a corresponding receiver for up to buffer length values... not really needed here because we're always gonna be receiving, but seems like best practice. In go by example's words: "By default channels are unbuffered, meaning that they will only accept sends (chan <-) if there is a corresponding receive (<- chan) ready to receive the sent value. Buffered channels accept a limited number of values without a corresponding receiver for those values." Not accept means block...

Reply:
Cool, this all seems reasonable, thanks for the explanation 👍 _opens 'go by example' and gets reading_